code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -44,7 +44,7 @@
 /obj/item/weapon/storage/backpack/holding/handle_item_insertion(obj/item/W, prevent_warning = 0, mob/user)
 	if(istype(W, /obj/item/weapon/storage/backpack/holding) && !W.crit_fail)
 		var/safety = alert(user, "You feel this may not be the best idea.", "Put in [name]?", "Proceed", "Abort")
-		if(safety == "Abort" || !in_range(src, user) || !src || !W)
+		if(safety == "Abort" || !in_range(src, user) || !src || !W || user.incapacitated())

```

Comment:
@Miauw62 That just plain breaks the proc in general. They won't combine anymore. I'll revert it and see if it works normally.